src/client_reqwest.rs

```diff
@@ -66,7 +71,9 @@ impl From<reqwest::Error> for Error {
     }
 }
 
-#[async_trait]
+// Wasm target need not be `Send` because it is single-threaded
+#[cfg_attr(feature = "async-http-client-wasm", async_trait(?Send))]
+#[cfg_attr(not(feature = "async-http-client-wasm"), async_trait)]
```

Comment:
Why not simply based on `target_arch` rather than some feature? (as it's also done in `trait_async.rs`)

```suggestion
#[cfg_attr(target_arch = "wasm32", async_trait(?Send))]
#[cfg_attr(not(target_arch = "wasm32"), async_trait)]
```